src/applications/disability-benefits/686/config/form.js

```diff
@@ -121,82 +124,55 @@ const formConfig = {
     vaFileNumber
   },
   chapters: {
-    veteranInformation: {
-      title: 'Veteran Information',
+    applicantInformation: {
+      title: 'Applicant Information',
       pages: {
-        veteranInformation: {
-          path: 'veteran-information',
-          title: 'Veteran Information',
+        applicantInformation: {
+          title: 'Applicant Information',
+          path: 'applicant-information',
           uiSchema: {
-            veteranFullName: fullNameUI,
-            veteranSocialSecurityNumber: _.merge(ssnUI, {
-              'ui:required': form => !form.veteranVAfileNumber
-            }),
-            veteranVAfileNumber: {
-              'ui:title': 'VA file number (must have this or a Social Security number)',
-              'ui:required': form => !form.veteranSocialSecurityNumber,
-              'ui:help': VAFileNumberDescription,
-              'ui:errorMessages': {
-                pattern: 'Your VA file number must be between 7 to 9 digits'
-              }
-            },
-            'view:relationship': {
+            'ui:description': applicantDescription,
+            claimantFullName: fullNameUI,
+            claimantSocialSecurityNumber: ssnUI,
+            dateOfBirth: currentOrPastDateUI('Date of birth'),
+            relationshipToVet: {
```

Comment:
We should change this to `view:relationshipToVet`, since it's not in the backend schema.